portfolio/pom.xml

```diff
@@ -23,6 +23,16 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+    <dependency>
```

Comment:
Shelby put a blank line here to separate each dependency block. I don't know what Google rules say about POM files but it seemed easier to read that way. No need to change this at this time but consider space separating the dependency blocks at your convenience.

Reply:
Originally, I was thinking to group all dependencies together under the dependencies tag separated by a blank line from other groups ike properties, and the individual dependency tags are together under the big group. But I see how it makes it hard to distinguish the individual dependency, I will add a blank line between each pair of depndencies.